src/navigation/tabnav.scss

```diff
@@ -7,12 +7,16 @@
 }
 
 .tabnav-tabs {
+  display: flex;
   // stylelint-disable-next-line primer/spacing
   margin-bottom: -1px;
+  overflow: auto;
 }
 
 .tabnav-tab {
-  display: inline-block;
+  display: flex;
+  align-items: center;
```

Comment:
```suggestion
  display: inline-block;
```

Not sure if this is needed? Adding flexbox would remove the "natural" whitespace. Here an example https://github.com/primer/css/network/updates:

![Screen Shot 2020-07-08 at 22 14 13](https://user-images.githubusercontent.com/378023/86923022-714f3f80-c168-11ea-9bc5-83fbea3780c1.png)